.eslintignore

```diff
@@ -0,0 +1,6 @@
+dist
+node_modules
+.github
+.changeset
+*.md
```

Comment:
Do we need this? I tried removing it and seemed to run fine — I assume ESLint ignores Markdown files by default?

```suggestion
```

Reply:
You're right. It is not necessary. I copied it from `.prettierignore` and forgot to remove it.